examples/log-ingestion/log_ingestion_demo_guide.md

```diff
@@ -1,121 +1,82 @@
-Go [here](https://opensearch.org/downloads.html) to do so before moving on to the next step of this guide.
-
-If Data Prepper is running correctly, you should see something similar to the following line as the latest output in your terminal.
+4. Now that you understand a bit more about how FluentBit and OpenSearch are set up, run them with:
 
 ```
-INFO  com.amazon.dataprepper.pipeline.ProcessWorker - grok-pipeline Worker: No records received from buffer
+docker-compose up
 ```
 
-### FluentBit Setup
+### Data Prepper Setup
 
-The FluentBit setup includes a `docker-compose.yaml`, a `fluent-bit.conf`, and a log file which corresponds to where FluentBit will look for logs.
+1. Build the Data Prepper 1.2 SNAPSHOT Docker image by following the instructions found [here](../../release/docker/README.md).
 
-1. Create the following `docker-compose.yaml`. This `docker-compose.yaml` will pull the FluentBit Docker image, and will mount your `fluent-bit.conf` and log file to this Docker image. In this case, the log file is named `test.log`.
+ 
+2. Take a look at [log_pipeline.yaml](log_pipeline.yaml). This configuration will take logs sent to the [http source](../../data-prepper-plugins/http-source), 
+process them with the [Grok Prepper](../../data-prepper-plugins/grok-prepper) by matching against the `COMMONAPACHELOG` pattern, 
+and send the processed logs to a local [OpenSearch sink](../../data-prepper-plugins/opensearch) to an index named `apache_logs`.
 
-```yaml
-version: "3.7"
 
-services:
-  fluent-bit:
-    image: fluent/fluent-bit
-    volumes:
-      - ./fluent-bit.conf:/fluent-bit/etc/fluent-bit.conf
-      - ./test.log:/var/log/test.log
+3. Run the Data Prepper docker image with the `log_pipeline.yaml` from step 2 passed in. This command attaches the Data Prepper Docker image to the Docker network `log-ingestion_opensearch_net` so that 
+FluentBit is able to send logs to the http source of Data Prepper.
+
+```
+docker run --name data-prepper -v /full/path/to/log_pipeline.yaml:/usr/share/data-prepper/pipelines.yaml --network "log-ingestion_opensearch-net" opensearch-data-prepper:1.2.0-SNAPSHOT
 ```
 
-2. Create the following `fluent-bit.conf`. This config will tell FluentBit to tail the `/var/log/test.log` file for logs, and uses the FluentBit http output plugin to forward these logs to the http source of Data Prepper, which runs by default on port 2021.
+If Data Prepper is running correctly, you should see something similar to the following line as the latest output in your terminal.
 
 ```
-[INPUT]
-  name                  tail
-  refresh_interval      5
-  path                  /var/log/test.log
-  read_from_head        true
-
-[OUTPUT]
-  Name http
-  Match *
-  Host host.docker.internal
-  Port 2021
-  URI /log/ingest
-  Format json
+INFO  com.amazon.dataprepper.pipeline.ProcessWorker - log-pipeline Worker: No records received from buffer
 ```
 
```

Comment:
Consider adding a command like `echo '[{"log": "sample log"}]' >> test.log` to verify the pipeline is working.

Reply:
I can add something like this as an additional option to both copying the sample logs in or using the apache log generator.